.pre-commit-config.yaml

```diff
@@ -11,6 +11,12 @@ repos:
         exclude: '^(tests/.*\.exp|tests/cmdline/.*|tests/.*/data/.*|ports/espressif/esp-idf-config/.*|ports/espressif/boards/.*/sdkconfig)'
     -   id: trailing-whitespace
         exclude: '^(tests/.*\.exp|tests/cmdline/.*|tests/.*/data/.*|lib/mbedtls_errors/.*)'
+-   repo: https://github.com/codespell-project/codespell
+    rev: v2.2.4
+    hooks:
+    -   id: codespell
+        #args: [-w]
+        exclude: ^(locale/|lib/)
```

Comment:
exclude local since it not English, and lib since it is mostly 3rd party code.

Reply:
Can you run this again locally to remove the lib/  entries that are in changed files?